docs/installing-solidity.rst

```diff
@@ -60,17 +60,33 @@ Please refer to the solc-js repository for instructions.
 Docker
 ======
 
-We provide up to date docker builds for the compiler. The ``stable``
-repository contains released versions while the ``nightly``
-repository contains potentially unstable changes in the develop branch.
+Docker images of Solidity builds are available. Use the ``stable`` tag for the
+latest released version, and ``nightly`` for potentially unstable changes in the develop branch.
+
+.. code-block:: bash
+
+    docker run ethereum/solc:stable
```

Comment:
Perhaps already show how to use options here? I think we should explain what this example code actually does - currently we only explain how to specify the version, so it might be better to combine it with what follows below. What do you think?

Reply:
What about `docker run ethereum/solc:stable --help`?